player/models.py

```diff
@@ -49,6 +50,11 @@ def validate_and_save(self, birth_date, favorite_ball_game):
         else:
             self.favorite_ball_game = favorite_ball_game
 
+        if profile_picture:
+            try:
+                self.profile_pic.save(profile_picture.name, profile_picture, save=False)
```

Comment:
someone can upload an image with an already existing image and it will change the profile picture for a different player :P 

you can do something like creating a random string and using the user-name to hash the picture name

small example:

```
def user_directory_path(instance, filename):

	# file will be uploaded to MEDIA_ROOT / user_<id>/<filename>
	return 'user_{0}/{1}'.format(instance.user.id, filename)

class MyModel(models.Model):
	upload = models.ImageField(upload_to = user_directory_path)

```

Reply:
thanks, your right, working on it now

Reply:
done.